### PR TITLE
added exportgroth16 call and moved genData

### PR DESCRIPTION
**Pull request by Meyanis95**
## Motivation

Add util function to call Verifier>sol smart contract

## Change Summary

- Added `exportCallDataGroth16()` to utils to prepare callData to the smart contract
- Moved `genData()` to pcd utils as it is shared by all the test scripts
- Added `gen-pdf` to the monorepo commands

## Merge Checklist

_Choose all relevant options below by adding an `x` now or at any time before submitting for review_

- [X] PR title adheres to the [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/) standard
- [X] PR has a [change set](https://github.com/privacy-scaling-explorations/anon-aadhaar/CHANGELOG.md)
- [X] PR has been tagged with a change label(s) (i.e. documentation, feature, bug fix, or chore)
- [X] PR includes documentation if necessary.

## Additional Context

If this is a relatively large or complex change, provide more details here that will help reviewers

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,5 +50,5 @@ yarn test:libraries
 Generate pdf file with pkcs#1 schema. This action will create pdf file name `signed.pdf` signed by `certificate.cer` in `packages/anon-aadhaar-pcd/build` folder.
 
 ```bash
-yarn pdf:pcd
+yarn gen-proof:pcd
```
Comment (vuvoth) [inline README.md:53]: I think this should be pdf:pcd

Comment (vuvoth) [reply README.md:53]: Should add instruction to gen proof? :D

**Review by vuvoth — APPROVED** [review]
I leaf few comments. Please check it! 
Everything else is make sense. I will approve first.

File: README.md (hunk 1 above)
Comment (Meyanis95) [reply README.md:53]: Yes sorry I mixed things up.
I fixed the typo 
Thanks :)

**Pull request merged**